python/ray/workers/default_worker.py

```diff
@@ -1,155 +1,18 @@
-    else:
-        raise ValueError("Unknown worker type: " + args.worker_type)
-
-    # NOTE(suquark): We must initialize the external storage before we
-    # connect to raylet. Otherwise we may receive requests before the
-    # external storage is intialized.
-    if mode == ray.RESTORE_WORKER_MODE or mode == ray.SPILL_WORKER_MODE:
-        from ray import external_storage
-        if args.object_spilling_config:
-            object_spilling_config = base64.b64decode(
-                args.object_spilling_config)
-            object_spilling_config = json.loads(object_spilling_config)
-        else:
-            object_spilling_config = {}
-        external_storage.setup_external_storage(object_spilling_config)
-
-    raylet_ip_address = args.raylet_ip_address
-    if raylet_ip_address is None:
-        raylet_ip_address = args.node_ip_address
-
-    code_search_path = args.code_search_path
-    if code_search_path is not None:
-        for p in code_search_path.split(":"):
-            if os.path.isfile(p):
-                p = os.path.dirname(p)
-            sys.path.append(p)
-
-    ray_params = RayParams(
-        node_ip_address=args.node_ip_address,
-        raylet_ip_address=raylet_ip_address,
-        node_manager_port=args.node_manager_port,
-        redis_address=args.redis_address,
-        redis_password=args.redis_password,
-        plasma_store_socket_name=args.object_store_name,
-        raylet_socket_name=args.raylet_name,
-        temp_dir=args.temp_dir,
-        load_code_from_local=args.load_code_from_local,
-        metrics_agent_port=args.metrics_agent_port,
-    )
-
-    node = ray.node.Node(
-        ray_params,
-        head=False,
-        shutdown_at_exit=False,
-        spawn_reaper=False,
-        connect_only=True)
-    ray.worker._global_node = node
-    ray.worker.connect(node, mode=mode)
-
-    # Redirect stdout and stderr to the default worker interceptor logger.
-    # NOTE: We deprecated redirect_worker_output arg,
-    # so we don't need to handle here.
-    stdout_interceptor = StandardStreamInterceptor(
-        setup_and_get_worker_interceptor_logger(is_for_stdout=True),
-        intercept_stdout=True)
-    stderr_interceptor = StandardStreamInterceptor(
-        setup_and_get_worker_interceptor_logger(is_for_stdout=False),
-        intercept_stdout=False)
-    with redirect_stdout(stdout_interceptor):
-        with redirect_stderr(stderr_interceptor):
-            if mode == ray.WORKER_MODE:
-                ray.worker.global_worker.main_loop()
-            elif (mode == ray.RESTORE_WORKER_MODE
-                  or mode == ray.SPILL_WORKER_MODE):
-                # It is handled by another thread in the C++ core worker.
-                # We just need to keep the worker alive.
-                while True:
-                    time.sleep(100000)
-            else:
-                raise ValueError(f"Unexcepted worker mode: {mode}")
-
+from ray.ray_logging import (StandardStreamInterceptor,
```

Comment:
Please add a comment in this file saying that adding functions causes issues with tensorflow (and any more detail you have about why)